java/operator/org.eclipse.theia.cloud.operator/src/main/java/org/eclipse/theia/cloud/operator/handler/session/LazySessionHandler.java

```diff
@@ -444,6 +446,16 @@ protected void createAndApplyDeployment(String correlationId, String sessionReso
         }
         K8sUtil.loadAndCreateDeploymentWithOwnerReference(client.kubernetes(), client.namespace(), correlationId,
                 deploymentYaml, Session.API, Session.KIND, sessionResourceName, sessionResourceUID, 0, deployment -> {
+
+                    LOGGER.info("Setting pod labels");
```

Comment:
I think this should only be a debug level log as nothing exiting or too relevant happens here functionality wise.
```suggestion
                    LOGGER.debug("Setting pod labels");
```